### ISSUE TITLE
add R documentation

### ISSUE THREAD

**Issue by ksonda** [internetofwater/docs.geoconnex.us#31]
I'm working on this here. https://gist.github.com/ksonda/5ae70a069a35ebe8ef4bd622df73530d

Rendering here for now https://kyleonda.quarto.pub/using-the-geoconnex-reference-feature-server-with-r/

I would like the python work to be more aligned. Theres a few places where I'm not sure the workflows are quite the same. 

Im not really a jupytr notebook person so if we need to maintain these links out I'm ok with it.

**Comment by ksonda** [comment]
I'm done. @C-Loftus @webb-ben  I'd like the python tutorial to be more or less aligned with the R, using the same examples and whatnot and structured the same way. I also think the organization in the docs.geoconnex.us needs to be aligned somewhat differently as well. Maybe remove the distinction between "reference features" and "full graph". And make it use-case aligend as I've done in the R doc. or something. @adplincinst take a look as well. @dblodgett-usgs  maybe jsut check the gist for R-goodness.

**Comment by dblodgett-usgs** [comment]
Looks great!! Thanks for the thoughtful preparation.

In addition to needing a copy edit, two comments:

1. you could mention that nhdplusTools has a geoconnex client: https://doi-usgs.github.io/nhdplusTools/reference/get_geoconnex_reference.html -- I think you should keep the base R / sf documentation of reference features, but mentioning that there is a more facilitated option available may be worth while for the less R-savvy user base.
2. I agree with use case alignment -- can we maybe take that one step further and give an example that is more relatable than the generic spatial / related data use case that's presented? 


```
    Finding identifiers and spatial geometries for real-world features

For example, discovering Dams and stream gage sites within an area of interest. 

    Finding features that are hydrologically related to a feature of interest (if that information is available)

For example, discovering stream gage sites along a given river.

    Finding datasets related to a feature of interest

For example, discovering stream temperature data related to a given river.
```

**Comment by ksonda** [comment]
This is a decent structure. @adplincinst added that int he part about attribute querying, providing a reference for what comparitors/ functionsa re available. Could reference https://portal.ogc.org/files/96288 and https://docs.geoserver.org/stable/en/user/tutorials/cql/cql_tutorial.html as an additional resource

**Comment by ksonda** [comment]
closed by #32
